data_ocean/filters.py

```diff
@@ -1,7 +1,25 @@
 import django_filters
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+from rest_framework.exceptions import ValidationError
+
 from .models import Register
 
 
+class ValidationBooleanField(forms.NullBooleanField):
+
+    def validate(self, value):
+        if str(value) not in ('true', 'false', '1', '0', 'True', 'False'):
+            raise ValidationError(
+                {self.label:
+                     _('Invalid request parameter: could be True, False, true, false, 1, 0')}
+            )
```

Comment:
add current  invalid value for message
```suggestion
            raise ValidationError({
                self.label: _(f'Invalid request parameter: "{value}", could be True, False, true, false, 1, 0'),
            })
```

Reply:
value in form already is cleaned. So moved the whole process to a custom widget again